README.md

```diff
@@ -18,10 +18,27 @@ Then add **hubot-heroku-keepalive** to your `external-scripts.json`:
 
 ## Configuring
 
-hubot-heroku-keepalive is configured by two environment variables:
```

Comment:
I think you can just remove the quantifier for future proofing. It reads the same as "is configured by environment variables:"